kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -0,0 +1,72 @@
+package com.kodilla.testing.shape;
+
+import com.kodilla.testing.com.kodilla.testing.shape.*;
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ShapeCollectorTestSuite {
+    private static int counter = 1;
+
+    @Before
+    public void Before() {
+        System.out.println("Start testu nr " + counter);
+    }
+
+    @After
+    public void After() {
+        System.out.println("Koniec testu nr " + counter);
+        counter++;
+    }
+
+    @Test
+    public void testAddFigure() {
+        //given
+        ShapeCollector objShapeCollector = new ShapeCollector();
+        //when
+        Shape circle = new Circle(2.0);
+        boolean result = objShapeCollector.addFigure(circle);
+        //then
+        Assert.assertTrue(result);
+    }
+
+    @Test
+    public void testRemoveFigure() {
+        //given
+        ShapeCollector objShapeCollector = new ShapeCollector();
+        Shape circle = new Circle(2.0);
+        objShapeCollector.addFigure(circle);
+        //when
+        boolean result = objShapeCollector.removeFigure(circle);
+
+        //then
+        Assert.assertTrue(result);
+        Assert.assertEquals(0,objShapeCollector.shapesListSize());
```

Comment:
you don't have test for case when you try to remove figure which does not exist in ShapeCollector